### PR TITLE
Clean up: validation test

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions
- establish alphabetical order
- fixed indentation
- renamed `experimentalTechnique` to `technique` (to reduce list of properties names because `technique` is used for datasetVersion in a similar context)

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

File: schemas/validationTest.schema.tpl.json [hunk 1]
```diff
@@ -1,60 +1,60 @@
...
+    "studyTarget": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add all study targets of this validation test.",
+      "_linkedCategories": [
+        "studyTarget"
+      ]
+    },
+    "technique": {
```
Comment (apdavison) [inline schemas/validationTest.schema.tpl.json]: I'm not sure about this renaming, it is ambiguous. In the context of a validation test, "technique" could refer to the data analysis pipeline used for the test or to the experimental technique used to obtain the reference data. The original name, "experimentalTechnique" disambiguates this.

Comment (UlrikeS91) [reply schemas/validationTest.schema.tpl.json]: I think that is a valid point. @lzehl or @Peyman-N any objections?

Comment (Peyman-N) [reply schemas/validationTest.schema.tpl.json]: Me too, I think it should be `experimentalTechnique`

Comment (lzehl) [reply schemas/validationTest.schema.tpl.json]: @apdavison and @Peyman-N I need more clarification : Generally I see the point not using just "technique" then, but I'm also wondering why there is no option in validationTest to refer to a data analysis pipeline? Some more questions to better understand the context: do you specifically want to ask for the data acquisition technique? and why do you want to ask for it? what importance does the original technique used to obtain the reference data have for a validation test?

Comment (apdavison) [reply schemas/validationTest.schema.tpl.json]: Ideally all reference data would be part of a Dataset, and so have full contextual metadata. Often, however, reference data is just taken from a published article (e.g. summary statistics), in that case the experimental technique gives us at least some (minimal) context.

Its value is largely for searching/sorting/filtering, for example we'd like experimentalists who are expert in a given technique to comment on validation tests, point out any limitations of the reference data, etc.

Comment (lzehl) [reply schemas/validationTest.schema.tpl.json]: @apdavison thank you for the clarification. I'm not sure I like the "experimentalTechnique" as property name. Is the acquisition technique of the data sufficient for you in this context? Then I would perfer "dataAcquisitionTechnique" OR "datasetAcquisitionTechnique" or "testDataAcquisition" or something like this... what do you think?

**Review by lzehl — APPROVED** [review]
@apdavison and @Peyman-N since I did not come up with a better name I've changed it back to experimentalTechnique. Please double check the update and merge if you also approve now.

**Comment by lzehl** [comment]
change to "referenceDataAcquisition"

**Review by lzehl — APPROVED** [review]
@apdavison AND @Peyman-N please double check the updated schema:
+ I've changed **experimentalTechnique** to **referenceDataAcquisition**
+ I've also changed the value of **referenceDataAcquisition** to be an array (in case multiple acquisition techniques need to be specified)

Feel free to merge if you approve the changes

**Pull request merged**